components/Notes.tsx

```
@@ -49,16 +49,25 @@ const Notes = ({ isOpen, onClose, userId }: { isOpen: boolean; onClose: () => vo
         aria-labelledby="notes-modal-title"
+        onClick={(e) => e.stopPropagation()}
       >
         <div className="p-4 flex justify-between items-center border-b border-white/10">
           <h2 id="notes-modal-title" className="text-lg font-bold">Notes</h2>
-          <button onClick={onClose} aria-label="Close Notes">
+          <button
+            onClick={onClose}
+            aria-label="Close Notes"
+            title="Close (Esc)"
+            className="rounded-full p-1 hover:bg-white/10 transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)